### PR TITLE
ci(typescript): update readme.md, license.md and dependency-review.yml

### LINKED ISSUE #95: Update license.md and readme.md file and dependency-review.yml file.
- MIT License is added
- Dependabot is set up
- Dependency Review is set up

### PR DESCRIPTION
**Pull request by MaryamAdnan3**
- [x] MIT License is added
- [x] Dependency Review is set up.
- [x] ReadMe is refactored.

Closes #95

### CODE REVIEW

**Review by usamabintariq — DISMISSED** [review]

**Review by wajahatiqbal — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -9,11 +9,13 @@ They provide common runtime utilities needed by SDKs to make API calls and handl
...
 | [@apimatic/core](packages/core) | Provides core http logic, file wrapper, api response and validation error classes. |
-
-## License
-
-© 2020 APIMatic Ltd. All Rights Reserved.
-
-This library is meant to be used in SDKs generated by APIMatic Code Generator. Please get in
-touch with our [support](mailto:mehdi@apimatic.io) for more information on licensing.
+| [@apimatic/core-interfaces](packages/core-interfaces) | Provides common interfaces to core. |
+| [@apimatic/file-wrapper](packages/file-wrapper) | Provides file wrapper support to core. |
```
Comment (wajahatiqbal) [inline README.md:17]: Wrapper around the file types and their utilities.

File: README.md [hunk 2]
```diff
@@ -9,11 +9,13 @@ They provide common runtime utilities needed by SDKs to make API calls and handl
...
+| [@apimatic/http-headers](packages/http-headers) | Provides http hedaers support to core. |
```
Comment (wajahatiqbal) [inline README.md:18]: HTTP Headers utilities for apimatic-js-runtime libraries.

File: README.md [hunk 3]
```diff
@@ -9,11 +9,13 @@ They provide common runtime utilities needed by SDKs to make API calls and handl
...
+| [@apimatic/http-query](packages/http-query) | Provides http query support to core. |
```
Comment (wajahatiqbal) [inline README.md:19]: HTTP Query utilities for apimatic-js-runtime libraries

File: README.md [hunk 4]
```diff
@@ -9,11 +9,13 @@ They provide common runtime utilities needed by SDKs to make API calls and handl
...
+| [@apimatic/oauth-adapters](packages/oauth-adapters) | Provides oauth schemes pluggables. |
```
Comment (wajahatiqbal) [inline README.md:20]: Provides pluggable adapters for OAth 2.0 authentication schemes.

File: README.md [hunk 5]
```diff
@@ -9,11 +9,13 @@ They provide common runtime utilities needed by SDKs to make API calls and handl
...
+| [@apimatic/xml-adapter](packages/xml-adapter) | Provides xml serialization and deserialization support to core. |
```
Comment (wajahatiqbal) [inline README.md:21]: Provides XML serialization and deserialization utilities for apimatic-js-runtime libraries.

**Review by wajahatiqbal — CHANGES_REQUESTED** [review]

**Review by usamabintariq — APPROVED** [review]

**Review by wajahatiqbal — APPROVED** [review]

**Review by usamabintariq — APPROVED** [review]

**Review by wajahatiqbal — APPROVED** [review]

**Pull request merged**